lib/benchmarks/initial-render.bench.tsx

```diff
@@ -113,6 +114,32 @@ function ReactHookFormInitialRenderBenchmark() {
   );
 }
 
+function ReactFinalFormInitialRenderBenchmark() {
+
+  return (
+    <Form
+      onSubmit={onSubmit}
+      validate={validate}
+      render={({ handleSubmit, name }) => (
+        <form onSubmit={handleSubmit}>
```

Comment:
Let's remove the `form` and have it be a `<>` instead, like the other benchmarks